components/connector/internal/gqlschema/schema.graphql

```diff
@@ -0,0 +1,59 @@
+# Token
+type Token {
+    url: String! # eg.: "https://connector-service.test.cluster.kyma.cx/v1/applications/info?token=1edfc34g"
+    token: String! # eg.: "1edfc34g"
+}
+
+# Certificate
+type Certificate {
```

Comment:
I would go with full names:

```graphql
certificate: String!
certificateAuthority: String!
clientCertificate: String!
```

Reply:
I changed it slightly, please take a look